roles/robot-pkgs/tasks/main.yml

```diff
@@ -24,8 +28,8 @@
     dest: '{{ global_profile_path }}'
     marker: "## {mark} ROS profile entries ##"
     block: |
-      source /opt/ros/kinetic/setup.bash
-      export TURTLEBOT_3D_SENSOR=kinect
+      source /opt/ros/melodic/setup.bash
```

Comment:
Can we add a variable called `ros_release` and set it to `melodic`?

Reply:
removed with commit 5bb24a2

Reply:
Again, a great idea so we only have to type the release name once. See commit f8e93cb.